pkg/types/fnref.go

```diff
@@ -2,18 +2,24 @@ package types
 
 import (
 	"errors"
-	"fmt"
 	"regexp"
+
+	metav1 "k8s.io/apimachinery/pkg/apis/meta/v1"
 )
 
 const (
 	RuntimeDelimiter = "://"
-	groupRuntime     = 1
-	groupRuntimeID   = 2
+)
+
+const (
+	groupFnRef = iota
```

Comment:
I know it wasn't there before either, but could you add a comment above this group enum to explain the purpose of them? maybe: `// Capture groups for the FnRef regex`